### ISSUE TITLE
Accessibility issues on blog posts

### ISSUE THREAD

**Issue by kinueng** [OpenLiberty/openliberty.io#328]
Dynamic assessment plugin (DAP) is reporting some accessibility violations on the blog posts.

Some errors range from needing alt text on images to aria labels.  The errors appear on elements that are shared across all the posts, like the Twitter icon/button.

Here is a sample from this blog post
https://openliberty.io/blog/2018/03/22/distributed-in-memory-session-caching.html

![image.png](https://images.zenhubusercontent.com/59d7b56bb0222d5de47a9368/6765b1b8-8d00-482e-b40f-c7390a1b0d91)
